### PR TITLE
SLING-10642 update unittests

### PR DESCRIPTION
**Pull request by joerghoh**

### CODE REVIEW

**Comment by kwin** [comment]
Shouldn't we try to remove usage of commons-testing alltogether? The only remaining usages are in `JcrNodeResourceIteratorTest` leveraging `MockNode` and `MockNodeIterator` which should be replaced by https://sling.apache.org/documentation/development/jcr-mock.html.

File: pom.xml [hunk 1]
```diff
@@ -260,6 +260,12 @@
             <version>1.4</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```
Comment (kwin) [inline pom.xml:263]: This dependency is already declared in line 246

**Comment by kwin** [comment]
I added one more commit in https://github.com/apache/sling-org-apache-sling-jcr-resource/commit/da4e5c044a25b5bf2c48cb666f1d2f7d6faa1e1a to get rid of commons.testing. Feel free to cherry pick to your PR.

**Comment by joerghoh** [comment]
I merged da4e5c044a25b5bf2c48cb666f1d2f7d6faa1e1a, thanks for your help.

**Review by kwin — APPROVED** [review]

**Pull request merged**